from shared.domain_exception import DomainException

# FIXME: specification must stop at domain.
# TODO: validation_exception, not_found_exception


class ValueObjectException(DomainException):
    pass
